packages/headless/src/controllers/context/headless-context.ssr.ts

```diff
@@ -1,5 +1,6 @@
 import {SearchEngine} from '../../app/search-engine/search-engine.js';
```

Comment:
This is the non-commerce context controller, I believe we should also do the same thing in the commerce context controller.

Reply:
Also we should adjust the tests (and in the case of the commerce SSR controller, add tests).